src/hooks/useMobileDetect.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function useMobileDetect() {
```

Comment:
@f1lander Are we introducing this useMobileDetect for the first time here?

Long term, and in relation to our plan for the mobile redesign, do you think this is approach we should ideally be taking?

Reply:
@crabsinger yes this is for the first time, just a simple hooks thats allow to us to detect if is a mobile device for this feature is important because I can decide to implement the clickaway listener so the "see more" option can be clickable because on touch device there is note onhover event to show the tooltip